### PR TITLE
Show field type when adding it to the content profile

### PR DESCRIPTION
**Pull request by pablopunk**
SDESK-3572

types

### CODE REVIEW

File: scripts/apps/workspace/helpers/getTypeForFieldId.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        case 'embed':
+            return gettext('embed');
+        case 'related_content':
+            return gettext('related content');
+        default:
+            return 'CV';
+    }
+};
+
+export const getTypeForFieldId = (fieldId, vocabularies: Array<IVocabulary>) => {
```
[the remark this answers is not in the document]
Comment (pablopunk) [reply scripts/apps/workspace/helpers/getTypeForFieldId.ts]: It was used, I removed it in a recent commit (that's why the PR is marked as WIP)

File: scripts/superdesk-interfaces/Vocabulary.ts [hunk 2]
```diff
@@ -16,7 +18,7 @@ export interface IVocabulary {
     priority: number;
     unique_field: string;
     schema: {};
-    field_type: string;
+    field_type: VocabularyFieldType;
```
[the remark this answers is not in the document]
Comment (pablopunk) [reply scripts/superdesk-interfaces/Vocabulary.ts]: Cool

File: scripts/apps/workspace/helpers/getTypeForFieldId.ts [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        case 'media':
+            return '';
+        case 'date':
+            return gettext('date');
+        case 'embed':
+            return gettext('embed');
+        case 'related_content':
+            return gettext('related content');
+        default:
+            return 'CV';
```
[the remark this answers is not in the document]
Comment (pablopunk) [reply scripts/apps/workspace/helpers/getTypeForFieldId.ts]: forgot the gettext but it should be `gettext('CV')` as requested in the task

File: scripts/apps/workspace/helpers/getTypeForFieldId.ts [hunk 4]
```diff
@@ -0,0 +1,30 @@
+import {VocabularyFieldType, IVocabulary} from 'superdesk-interfaces/Vocabulary';
+import {gettext} from 'core/ui/components/utils';
+
+// export type VocabularyFieldType = 'text' | 'media' | 'date' | 'embed' | 'related_content';
+export const getLabelForType = (fieldType: VocabularyFieldType) => {
+    switch (fieldType) {
+        case 'text':
+            return gettext('text');
+        case 'media':
+            return '';
```
[the remark this answers is not in the document]
Comment (pablopunk) [reply scripts/apps/workspace/helpers/getTypeForFieldId.ts]: As mentioned in the task, media types should not show anything as their type

File: scripts/apps/workspace/helpers/getTypeForFieldId.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (tomaskikutis) [reply scripts/apps/workspace/helpers/getTypeForFieldId.ts]: Check with Karel if you can change it to `gettext('custom vocabulary')`. The translators won't be able to translate "CV" and not using an abbreviation is easier to understand.

File: scripts/apps/workspace/helpers/getTypeForFieldId.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (tomaskikutis) [reply scripts/apps/workspace/helpers/getTypeForFieldId.ts]: Sorry, I didn't notice the WIP tag, will check again when it's removed.

File: scripts/apps/workspace/helpers/getTypeForFieldId.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (pablopunk) [reply scripts/apps/workspace/helpers/getTypeForFieldId.ts]: good point

File: scripts/apps/workspace/helpers/getTypeForFieldId.ts [hunk 5]
```diff
@@ -0,0 +1,29 @@
+import {IVocabulary} from 'superdesk-interfaces/Vocabulary';
+import {gettext} from 'core/ui/components/utils';
+
+export const getLabelForType = (fieldType: IVocabulary['field_type']) => {
+    switch (fieldType) {
```
Comment (petrjasek) [inline scripts/apps/workspace/helpers/getTypeForFieldId.ts]: I don't mind switch, but why not use a dict here?

**Pull request merged**